hoomd/md/methods.py

```diff
@@ -677,34 +684,72 @@ class Langevin(_Method):
     assumption is valid when underdamped: :math:`\frac{m}{\gamma} \gg \delta t`.
     Use `Brownian` if your system is not underdamped.
 
-    :py:class:`Langevin` uses the same integrator as :py:class:`NVE` with the
-    additional force term :math:`- \gamma \cdot \vec{v} + \vec{F}_\mathrm{R}`.
-    The random force :math:`\vec{F}_\mathrm{R}` is drawn from a uniform random
-    number distribution.
+    `Langevin` uses the same integrator as `NVE` with the additional force term
+    :math:`- \gamma \cdot \vec{v} + \vec{F}_\mathrm{R}`. The random force 
+    :math:`\vec{F}_\mathrm{R}` is drawn from a uniform random number 
+    distribution.
 
     You can specify :math:`\gamma` in two ways:
 
-    1. Use ``set_gamma()`` to specify it directly, with independent
-       values for each particle type in the system.
-    2. Specify :math:`\alpha` which scales the particle diameter to
-       :math:`\gamma = \alpha d_i`. The units of
-       :math:`\alpha` are mass / distance / time.
+    1. Specify :math:`\alpha` which scales the particle diameter to 
+       :math:`\gamma = \alpha d_i`. The units of :math:`\alpha` are 
+       mass / distance / time.
+    2. After the method object is created, specify the 
+       attribute ``gamma`` and ``gamma_r`` for rotational damping or random 
+       torque to assign them directly, with independent values for each 
+       particle type in the system.
 
-    *kT* can be a variant type, allowing for temperature ramps in simulation
-    runs.
-
-    A :py:class:`hoomd.compute.thermo` is automatically created and associated
-    with *group*.
 
     Warning:
         When restarting a simulation, the energy of the reservoir will be reset
         to zero.
 
+
     Examples::
 
-        all=hoomd.filter.All()
-        langevin = hoomd.md.methods.Langevin(filter=all, kT=0.2, seed=1, alpha=1.0)
-        integrator = hoomd.md.Integrator(dt=0.001, methods=[langevin], forces=[lj])
+        langevin = hoomd.md.methods.Langevin(filter=hoomd.filter.All(), kT=0.2, 
+        seed=1, alpha=1.0)
+        integrator = hoomd.md.Integrator(dt=0.001, methods=[langevin], 
+        forces=[lj])
+
+
+    Examples of using ``gamma`` or ``gamma_r`` on drag coefficient::
+
+        langevin = hoomd.md.methods.Brownian(filter=hoomd.filter.All(), kT=0.2,
+        seed=1)
+        langevin.gamma = 2.0
+        langevin.gamma_r = [1.0,2.0,3.0]
```

Comment:
These are type parameters so to set for all types change the default (i.e. `langevin.gamma.default = 2.0`).